playbooks/vars/defaults.yaml

```diff
@@ -138,13 +138,20 @@ update_local_pki: false
 sriov_services:
   - OS::TripleO::Services::NeutronSriovAgent
 
+ovsdpdk_services:
+  - OS::TripleO::Services::ComputeNeutronOvsDpdk
+  - OS::TripleO::Services::OvsDpdkNetcontrold
```

Comment:
This shouldn't be used, netcontrold is not used anymore and its features were added to native OVS.

Reply:
```suggestion
```